src/main/scala/com/itechart/ny_accidents/service/metric/PopulationMetricService.scala

```diff
@@ -18,8 +18,12 @@ class PopulationMetricService @Inject()(populationStorage: PopulationStorage) {
       populationMap.get(districtHash) match {
         case Some(value) =>
           val density = PopulationService.calculateDensity(value)
-          val ration = accidentsNumber / density
-          Some(district.districtName, ration, density, accidentsNumber)
+          val ration: Double = accidentsNumber / density
```

Comment:
Sounds like a generic function, that takes Double and returns Option[Double], that could be reused in various modules of the code

Reply:
Created in NumberUtils object